assets/gzip/methodology.md

```
@@ -0,0 +1,13 @@
+# Gzip methodology
+
+Ingestion stats only, no search  
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)